Query app update (#853)
graphprotocol/docs: website/pages/en/subgraphs/querying/from-an-application.mdx

> ## Getting GraphQL Endpoint

Reviewer: ```suggestion
## Getting GraphQL Endpoints
```

> Once a subgraph is deployed to [Subgraph Studio](https://thegraph.com/studio/) or [Graph Explorer](https://thegraph.com/explorer), you will receive a Network and Subgraph Studio endpoint for your GraphQL API that looks like this:

Reviewer: ```suggestion
Once a subgraph is deployed to [Subgraph Studio](https://thegraph.com/studio/) or [Graph Explorer](https://thegraph.com/explorer), you will receive two endpoints for your GraphQL API that look like this:
```

Reviewer: Or, better:
```suggestion
Once a subgraph is deployed to [Subgraph Studio](https://thegraph.com/studio/) or [Graph Explorer](https://thegraph.com/explorer), you will receive two GraphQL API endpoints that look like this:
```
